### PR TITLE
Configuration for Failed Test ReRun

### PR DESCRIPTION
**Pull request by sarangholey**
Hi Tom,

Please review the code for Failed Test Rerun configuration and suggest me any changes if required.

Thanks,
Sarang

### CODE REVIEW

File: src/test/java/com/qa/automation/runners/TestRunner.java [hunk 1]
```diff
@@ -8,10 +8,12 @@
 @CucumberOptions(
         features="classpath:features",//to tell cucumber where is ur feature file
         glue="com.qa.automation.stepdefs", // to tell cucumber where is ur step def code
-        tags="@ProdMainCategory", // to tell which tagged feature file to execute
```
Comment (visionjava7) [inline src/test/java/com/qa/automation/runners/TestRunner.java:11]: Please remove the tags, the "tags" should be empty.

**Review by visionjava7 — COMMENTED** [review]
Added the comments please check and revert it.

File: src/test/java/com/qa/automation/runners/TestRunner.java (hunk 1 above)
Comment (sarangholey) [reply src/test/java/com/qa/automation/runners/TestRunner.java:11]: Made the changes please check and approve the PR

Comment (visionjava7) [reply src/test/java/com/qa/automation/runners/TestRunner.java:11]: It Seems like the changes are good for now, approved.

**Pull request merged**